src/system-plugins/software-update-alert/public/js/software-update.js

```diff
@@ -68,7 +68,7 @@
     var logoTitleModel = {};
     logoTitleModel.cockpitVersion = ko.observable('N/A');
     logoTitleModel.bbSerial = ko.observable('N/A');
-    $('a.brand').attr('data-bind', 'attr: {title: "Version: " + cockpitVersion() + "\\nBB Serial: " + bbSerial()}');
+    $('html /deep/ a.brand').attr('data-bind', 'attr: {title: "Version: " + cockpitVersion() + "\\nBB Serial: " + bbSerial()}');
```

Comment:
yes and no.
there will always be a case where a plugin wan't to do something that is not covered by an extension point. Otherwise we have every element on the UI as an extension point.